### PR TITLE
Replace stopwords

### PR DESCRIPTION
**Pull request by Peritract**
This PR fixes the `replace_stopwords` function in preprocessing, and wires the various stopwords functions together in a more consistent manner.

`remove_stopwords` now calls `replace_stopwords` which applies `_replace_stopwords` to a series. Stop word removal is functioning as expected, as is stop word replacement with arbitrary symbols.

### CODE REVIEW

File: texthero/preprocessing.py [hunk 1]
```diff
@@ -182,12 +182,26 @@ def remove_whitespace(input: pd.Series) -> pd.Series:
     return input.str.replace("\xa0", " ").str.split().str.join(" ")
 
 
-def _remove_stopwords(text: str, words: Set[str]) -> str:
+def _replace_stopwords(text: str, words: Set[str], symbol: str = " ") -> str:
     """
-    Remove block of digits from text.
+    Remove words in list from a text, replacing them with a symbol.
```
Comment (Peritract) [inline texthero/preprocessing.py:187]:
```suggestion
    Remove words in a set from a string, replacing them with a symbol.
```

**Comment by jbesomi** [comment]
Hi Petriract! Thank you very much for your PR! Yes, indeed passing the docstring test is very annoing. I'm spending a lot of time debugging that generally..

For your reference, before pushing, you can verify that all tests (unittest + doctest) pass locally by executing './check.sh' from the scripts folder. The contributing.md should mention that but maybe is not enough clear; let me know if that's the case!

Regards,

**Comment by Peritract** [comment]
Hi @jbesomi; it's not the doctests fault - I was not checking my arguments carefully enough.

The tests are passing now.

**Comment by jbesomi** [comment]
Great; thank you!

**Comment by jbesomi** [comment]
btw; I will add you on the contributors list as soon as I will open my computer

**Comment by Peritract** [comment]
Thank you very much.

**Comment by jbesomi** [comment]
Done. Thank you for your _very_ appreciated help!

**Pull request merged**